dxr/indexers.py

```diff
@@ -224,6 +224,9 @@ def links(self):
 
             (sort order, heading, [(icon, title, href), ...])
 
+            File views will replace any {{line}} within the href with the
+            last-selected line number.
```

Comment:
No, actually when I was writing this I hadn't realized that we scrolled to the first selected line.